recipe/patches/0001-version-patch.patch

```diff
@@ -0,0 +1,25 @@
+From 10e8c4474e2ed9b87f04428d1f2e6f44cc42bf86 Mon Sep 17 00:00:00 2001
+From: Lorenzo Pirritano <lpirritano@anaconda.com>
+Date: Thu, 20 Mar 2025 21:52:52 +0100
+Subject: [PATCH] version-patch
+
```

Comment:
Can you add a brief description here of the patch file's purpose

Reply:
sure!

```suggestion
Patch pyproject.toml to use the dynamic version from the VERSION file rather than a git tag.

This is because setuptools-git-versioning will not detect the version from the GitHub archive
(not a repository, and therefore there is no tag).

The file named VERSION with the correct version is created in the script section of the recipe.

```